README.md

```diff
@@ -11,11 +11,22 @@ This is an example [Universal Registrar](https://github.com/decentralized-identi
 ## Build and Run (Docker)
 
 ```
-docker build -f ./docker/Dockerfile . -t exampleorg/uni-registrar-driver-did-example
-docker run -p 9080:9080 exampleorg/uni-registrar-driver-did-example
+docker build -f ./docker/Dockerfile . -t sphereon/uni-registrar-driver-did-factom
+docker run -p 9080:9080 sphereon/uni-registrar-driver-did-factom
```

Comment:
Suggest to rename it to uni-registrar-driver-did-nodejs-factom or something. 
We will be building the official one based on Java because of all the work in the Java Identity client and support for full Factom DIDs. Don't want this implementation to clash with a logical name for the official one

Reply:
Updated the name. To avoid any confusion, I was not going to open a PR to make this driver part of the public UniRegistrar. We can just do our own deployment and it gives us something to work with for BOA until the official one is finished.

Reply:
For docker you need registries for hosting the images. So it is not only the potential confusion for the official driver, it would cause troubles with our Docker repos